app/main.py

```diff
@@ -1,18 +1,19 @@
+def validate_func(value: float, min_value: float, max_value: float, name_argument: str) -> int | float:
+    if min_value <= value <= max_value:
+        return value
+    else:
+        raise ValueError(f"{name_argument} should be between {min_value} and {max_value}")
+
```

Comment:
The last reject, I swear :)
Note that if statement has return, it means you don't need else statement because after return python will not run the code after if statement even if there isn't else statement ;)

Reply:
I removed it.